<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-S5FPP8F" name="zadatak3" tests="2" failures="1" timestamp="15 May 2018 21:11:50 GMT" time="1.048" errors="0">
  <testcase name="testSquare" time="0.003" classname="unittesting.calculator.CalculatorTestSquare"/>
  <testcase name="testSquareRoot" time="1.03" classname="unittesting.calculator.CalculatorTestSquare">
    <failure type="org.testng.internal.thread.ThreadTimeoutException" message="Method org.testng.internal.TestNGMethod.testSquareRoot() didn&amp;apos;t finish within the time-out 1000">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method org.testng.internal.TestNGMethod.testSquareRoot() didn't finish within the time-out 1000
	at unittesting.calculator.Calculator.squareRoot(Calculator.java:27)
	at unittesting.calculator.CalculatorTestSquare.testSquareRoot(CalculatorTestSquare.java:36)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 7 stack frames]]>
    </failure>
  </testcase> <!-- testSquareRoot -->
</testsuite> <!-- zadatak3 -->
